vis_server/protocols/FileLoader.py

```diff
@@ -24,7 +24,17 @@ def list_variables(self, file_name):
         out = {}
         for vname in reader.variables:
             var = reader.variables[vname]
+
+            # Get a displayable name for the variable
+            if hasattr(var, 'long_name'):
+                name = var.long_name
+            elif hasattr(var, 'title'):
+                name = var.title
+            elif hasattr(var, 'id'):
```

Comment:
Fixed.  Actually I meant to fall back on the name of the variable in the file.  I'd prefer not to rely on cdms attributes being present.